### ISSUE TITLE
5th Release?

### ISSUE THREAD

**Issue by RPSource** [blasten/turn.js#486]
Does anyone know anything about this release?

I was googling a way to make my magazine more responsive cause it has some issues on smaller screens like macbooks and this was the 4th Google result:

http://turnjs.com/catalog/#page/1

**Comment by davbaron** [comment]
Yes, if you go to the author's (Emmanuel) Twitter page, you will find several 'announcements' and references to the 5th version either being complete, or just about complete. But that's all. It appears that he has simply dropped the project cold. He is essentially unresponsive to inquiries. That said, he apparently is allowing the 4th version to be completely 'free', though, as I pointed out to him, within the source code is a statement that says it must be purchased/licensed. Again, he has been unresponsive when prompted about this.  It's too bad...

**Comment by closdesign** [comment]
deleting comment. Has nothing to do with my issue now.

**Comment by pkanb7** [comment]
@RPSource I need your help mate. I clicked the link you posted and i am curious about the slider. In Steve Jobs example there are more thumbnail photos than in my example(34 images). I just want the slider bar to move with more width in order to match with the number of my photos. Currently the slider displays my thumbnail photos repeatedly for 6 times.

@davbaron So is it sure that the 4th release is completely free? The thing is that you can't charge a plugin/service when you can't offer full documentation on issues.

I appreciate your help!

**Comment by davbaron** [comment]
@pkanb7 - I don't think anyone is 'sure' about anything.  As I said above, when I contacted him last year and asked about this, he wrote that "...you can get it for free...".  Again - he stopped responding to anyone and has not updated the web site in any way that fully 'proves' that the 4th release really is 'free'.

**Comment by closdesign** [comment]
Yes, and if you really wanted to go get the 4th generation or even the 5th generation Turn,js you could always go view the source code of any of the demos on the Turn JS website or with that Catalog link @RPSource posted above.  This would be unethical of course, but if you wanted to tool around with it until an official release, there is always that option.

**Comment by closdesign** [comment]
Alright guys, it looks like the Turn JS 4 is free. He does not specify this but the "Download" button on the Turn.JS website now gets the TurnJS4.zip file directly instead of redirecting to the payment page.  So my assumption would be Turn JS 4 is now free. There is not stopping the download of the zip file.

**Comment by davbaron** [comment]
@closdesign  - Yes, the download is 'free', but as I've pointed out in the past, within the .js source code of the primary file, there is a reference to turnjs.com/license.txt.  Within _that_ file it is specified that you must pay all fees, at which point you would receive the unique 6 character license code.  Emmanuel needs to either update this license file to indicate 'completely free', or he needs to update the main .js file to remove the reference to the license file and indicate that the code is free to use.

**Comment by closdesign** [comment]
@davbaron Ahhhh, totally makes sense now..  I see what you mean.  We ordered V4 a few years ago, but I don't remember receiving a 6 character license code from the author. But then again, it has been a few years.  Yes I agree. @blasten needs to update the license if v4 is totally free.

**Comment by elelsg** [comment]
For commercial use is not free 
Get a commercial license:
http://www.turnjs.com/get
